docs/administration/segregation.md

```diff
@@ -67,7 +67,26 @@ To apply a default marking when creating a new entity or relationship, you can c
     Simply adding markings as default markings is insufficient to display the markings when creating an entity or relationship. You also need to enable default markings in the customization settings of an entity or relationship. For example, to enable default markings for a new report, navigate to "Settings > Customization > Report > Markings" and toggle the option to `Activate/Desactivate default values`.
 
 
-## Management of multiple markings
+### Maximum shareable marking definitions
+
+This configuration allows to define, for each type of marking definitions, until which level
+we allow to share data externally (via Public dashboard or file export).
+
+The marking definitions that can be shared by a group are the ones
+- that are allowed for this group
+- and whose order are inferior or equal to the order of the maximum shareable markings defined for each marking type.
+
+Note that users with the Bypass capability can share all the markings.
+
+> For now this max marking definitions is used only for Public dashboard.
```

Comment:
To harmonize the way we write docs, we should use this syntax:

!!! note “<tilte of the bloc>”

        <block>

See:
![image](https://github.com/OpenCTI-Platform/docs/assets/70956706/9d7c161b-b363-4ef2-aa14-f6e8ac67fc6f)
![image](https://github.com/OpenCTI-Platform/docs/assets/70956706/6e9084fd-3f01-410d-9fe7-4f9d3d437c86)